src/model/HeatExchangerDesiccantBalancedFlowPerformanceDataType1.hpp

```diff
@@ -0,0 +1,328 @@
+
+    bool setHumidityRatioEquationCoefficient6(double humidityRatioEquationCoefficient6);
+
+    bool setHumidityRatioEquationCoefficient7(double humidityRatioEquationCoefficient7);
+
+    bool setHumidityRatioEquationCoefficient8(double humidityRatioEquationCoefficient8);
+
+    bool setMinimumRegenerationInletAirHumidityRatioforHumidityRatioEquation(double minimumRegenerationInletAirHumidityRatioforHumidityRatioEquation);
+
+    bool setMaximumRegenerationInletAirHumidityRatioforHumidityRatioEquation(double maximumRegenerationInletAirHumidityRatioforHumidityRatioEquation);
+
+    bool setMinimumRegenerationInletAirTemperatureforHumidityRatioEquation(double minimumRegenerationInletAirTemperatureforHumidityRatioEquation);
+
+    bool setMaximumRegenerationInletAirTemperatureforHumidityRatioEquation(double maximumRegenerationInletAirTemperatureforHumidityRatioEquation);
+
+    bool setMinimumProcessInletAirHumidityRatioforHumidityRatioEquation(double minimumProcessInletAirHumidityRatioforHumidityRatioEquation);
+
+    bool setMaximumProcessInletAirHumidityRatioforHumidityRatioEquation(double maximumProcessInletAirHumidityRatioforHumidityRatioEquation);
+
+    bool setMinimumProcessInletAirTemperatureforHumidityRatioEquation(double minimumProcessInletAirTemperatureforHumidityRatioEquation);
+
+    bool setMaximumProcessInletAirTemperatureforHumidityRatioEquation(double maximumProcessInletAirTemperatureforHumidityRatioEquation);
+
+    bool setMinimumRegenerationAirVelocityforHumidityRatioEquation(double minimumRegenerationAirVelocityforHumidityRatioEquation);
+
+    bool setMaximumRegenerationAirVelocityforHumidityRatioEquation(double maximumRegenerationAirVelocityforHumidityRatioEquation);
+
+    bool
+      setMinimumRegenerationOutletAirHumidityRatioforHumidityRatioEquation(double minimumRegenerationOutletAirHumidityRatioforHumidityRatioEquation);
+
+    bool
+      setMaximumRegenerationOutletAirHumidityRatioforHumidityRatioEquation(double maximumRegenerationOutletAirHumidityRatioforHumidityRatioEquation);
+
+    bool setMinimumRegenerationInletAirRelativeHumidityforHumidityRatioEquation(
+      double minimumRegenerationInletAirRelativeHumidityforHumidityRatioEquation);
+
+    bool setMaximumRegenerationInletAirRelativeHumidityforHumidityRatioEquation(
+      double maximumRegenerationInletAirRelativeHumidityforHumidityRatioEquation);
+
+    bool setMinimumProcessInletAirRelativeHumidityforHumidityRatioEquation(double minimumProcessInletAirRelativeHumidityforHumidityRatioEquation);
+
+    bool setMaximumProcessInletAirRelativeHumidityforHumidityRatioEquation(double maximumProcessInletAirRelativeHumidityforHumidityRatioEquation);
+
+    //@}
+    /** @name Other */
+    //@{
+
+    boost::optional<double> autosizedNominalAirFlowRate();
+
+    boost::optional<double> autosizedNominalAirFaceVelocity();
+
+    void autosize();
+
+    void applySizingValues();
```

Comment:
That looks wrong to me. This object derives from AirToAirComponent which derives from HVACComponent. HVACComponent already has an autosize() on the public side that forwards to the Impl.
HVACComponent_Impl::autosize is virtual, so you just need to override on the Impl side.

Remove these two.